jpa/eclipselink.jpa.test.jse/src/org/eclipse/persistence/jpa/test/canonical/DomainInterface_.java

```diff
@@ -0,0 +1,13 @@
+package org.eclipse.persistence.jpa.test.canonical;
+
+import javax.annotation.Generated;
+import javax.persistence.metamodel.SingularAttribute;
+import javax.persistence.metamodel.StaticMetamodel;
+
+@Generated(value = "Simulated Generation", date = "2018-10-09T11:24:45")
```

Comment:
Try to avoid using try the `javax.annotation.processing.Generated` annotation unless this is specifically needed for the test (I don't think it is).

Reply:
Reason for this is that I'm trying to mimic the actual scenario at our end. We have a customized annotation processor that generates JPA canonical metamodel classes. You are (probably) right that the `javax.annotation.processing.Generated` here doesn't serve a (technical) purpose - it is more of a descriptive thing.

Reply:
beware of the difference between `javax.annotation.Generated` which is in your import, and `javax.annotation.processing.Generated` you are referring to in your comment - they have different requirements on what exactly to put on the compiler's module-path for JDK 9+

Reply:
I will take it out and push later today. Do you guys have any thoughts already on the actual test case and what it seems to be exposing?